wiki/Ranking_Criteria/en.md

```diff
@@ -36,87 +81,113 @@ All rules are exactly that: **rules**. They are **not** guidelines and may **not
 -   **No guest mapper should have more difficulties in the mapset than the creator, all modes together**. For collaboration mapsets, the uploader should take part in all [collab](/wiki/Glossary/#collab) difficulties. The person who should upload the map is the one who contributed most to it. Maps with an ambiguous set of difficulties such as collabs will be evaluated on a case-by-case basis to ensure that the submitting mapper is properly accounted for.
 
-### Metadata
+### Guidelines
+
+-   **The difficulty spread should be linear and reasonable.** Linear difficulty spread means your difficulties have a comparable gap in star rating between each other. A reasonable spread means the usage of [difficulty appropriate gameplay elements](/wiki/Difficulties). If your mapset does not have an Easy difficulty your Normal difficulty should follow the general guidelines for Easy difficulties. If your Insane difficulty is at or above a 5 star rating it is recommended to include another Insane level difficulty between Hard and the 5+ Insane.
+-   **Your difficulties should all end at the same spot.** Having a fully-mapped Normal/Hard and a half-mapped Easy just looks sloppy/lazy. A full Easy may look boring to you, but not to a player that can't handle the harder difficulties.
+
+## Metadata
+
+### Rules
 
 -   **Metadata must be consistent across all difficulties.** This is to ensure that searching is consistent and clean across a mapset.
 -   **Eastern artists must be written in the proper 'surname firstname' format** (e.g. Japanese/Chinese/Korean). For artists which have a preferred romanisation, use this one. If you're unsure of your artist's name, then check previously ranked maps, Google, Wikipedia, or ask a BN for help.
--   **Songs with Japanese titles must use the Modified Hepburn method of romanisation in the "Romanised Title" field.** As a non-unicode field, long vowels such as "おう" and "うう" should be romanised into "ou" and "uu" to avoid macrons. ([Refer to this link for more information](https://en.wikipedia.org/wiki/Hepburn_romanization#Features_of_Hepburn_romanization)). Loan words should be expressed using romanisation from the original language.
+-   **Songs with Japanese titles must use the Modified Hepburn method of romanisation in the `Romanised Title` field.** As a non-unicode field, long vowels such as `おう` and `うう` should be romanised into `ou` and `uu` to avoid macrons. ([Refer to this link for more information](https://en.wikipedia.org/wiki/Hepburn_romanization#Features_of_Hepburn_romanization)). Loan words should be expressed using romanisation from the original language.
```

Comment:
- remove the period before the "Refer to" link in round brackets
- change `Refer to this link for more information` to `Hepburn romanization` or similar
- also, the `#Features_of_Hepburn_romanization` of the link above doesn't exist ... anymore?